configure.ac

```diff
@@ -202,6 +202,7 @@ elif test "x${enable_dev_build}" = "xyes"; then
    if test "z$orig_cflags" = "z"; then
       AC_C_FLAG([-g3])
       AC_C_FLAG([-O0])
+      AC_C_FLAG([-Wshadow])
```

Comment:
Wrong place to add this; please put it a few lines down, below
`dnl always want these CFLAGS`

BTW, the difference between
```
AC_C_FLAG([-Wwrite-strings]) <--- HERE(A)
if test x"${enable_gcc_ultra_verbose}" = x"yes" ; then
  AC_C_FLAG([-Wcast-qual])   <--- HERE(B)
```
is that the warnings in place (A) should always be clean in CI (and CI fails if there is a new warning), while the warnings under (B) are "informative" and we don't want CI to be strict about them.

Reply:
Thank you! :-)